website/static/js/pages/support-page.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Code for interaction in the support page
+ */
+
+'use strict';
+
+var Raven = require('raven-js');
+var $ = require('jquery');
+var $osf = require('js/osfHelpers');
+
+$(document).ready(function(){
+    /**
+     * Toggle individual item with jQuery
+     * @param item {Object} jQuery object for the .support-item element
+     * @param turnOff {Boolean} true if we are closing and open item
+     * @private
+     */
+    function _toggleItem (item, turnOff) {
```

Comment:
How come ` _toggleItem ` takes a turnOff argument rather than actually toggling:
```
item.hasClass('open') ? item.removeClass('open').addClass('collapsed') : item.removeClass('collapsed').addClass('open')
```